airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -146,6 +147,14 @@ public ValidatingConfigPersistence withValidation() {
 
   @Override
   public <T> void writeConfig(final AirbyteConfig configType, final String configId, final T config) throws IOException {
+    final Map<String, T> configIdToConfig = new HashMap<>() {
+
+      {
+        put(configId, config);
+      }
+
+    };
+    writeConfigs(configType, configIdToConfig);
     database.transaction(ctx -> {
```

Comment:
Should the rest of this be removed? Looks like it's writing twice?

Reply:
Thanks for the catch, it was a mistake